### DISCUSSION TITLE
How to get "galaxy braing" achivement.

### DISCUSSION THREAD

**Discussion by ncdinosgit** [community/community#18729] [Other Feature Feedback, Questions, & Ideas] [answered]
https://github.blog/2022-06-09-introducing-achievements-recognizing-the-many-stages-of-a-developers-coding-journey/

**Comment by YJPark0421** [comment, marked as answer]
I could provide support link.
https://github.com/orgs/github-community/discussions/18725#discussioncomment-2962693

**Comment by Tran-Quyen** [comment]
It need to answer 2 discussions.

**Comment by danialtavakoli** [comment]
My answer to get this badge. Thank you.
